katcp/inspecting_client.py

```diff
@@ -593,7 +631,19 @@ def inspect_sensors(self, name=None, timeout=None):
         timeout : float or None, optional
             Timeout for sensors inspection, None for no timeout
 
-        TODO Return value
+        Returns
+        -------
+        Tornado future that resolves with:
+
+        request_changes : AttrDict of sets or None
+            Contanins sets of added/removed request names
```

Comment:
Updated in commit to follow. The text was partially replaced by docstrings written by @bmerry in PR #176